rest/http_rest_service.cc

```diff
@@ -272,12 +268,21 @@ init()
                const std::string & payload)
         {
             std::string requestId = this->getHttpRequestId();
-            HttpRestConnection restConnection(connection, requestId, this);
+            HttpRestConnection restConnection(connection,
+                                              std::move(requestId), this);
```

Comment:
Pas besoin de std::move si this->getHttpRequestId() est passé directement. Maintenant je ne sais pas si l'optimisation de la compilation fait la substitution automatiquement.